### PR TITLE
Merge release 3.4

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: cdap-ui/templates/apps/predefined/EventClassifier.json [hunk 1]
```diff
@@ -0,0 +1,252 @@
+{
+    "artifact": {
+        "name": "cdap-data-pipeline",
+        "version": "3.4.0-SNAPSHOT",
```
Comment (anwar6953) [inline cdap-ui/templates/apps/predefined/EventClassifier.json]: should bump

File: cdap-ui/templates/apps/predefined/LogsAggregateDataPipeline.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
+{
+    "artifact": {
+        "name": "cdap-data-pipeline",
+        "version": "3.4.0-SNAPSHOT",
```
Comment (anwar6953) [inline cdap-ui/templates/apps/predefined/LogsAggregateDataPipeline.json]: should bump

File: cdap-ui/templates/apps/predefined/NaiveBayesTrainer.json [hunk 3]
```diff
@@ -0,0 +1,125 @@
+{
+    "artifact": {
+        "name": "cdap-data-pipeline",
+        "scope": "SYSTEM",
+        "version": "3.4.0-SNAPSHOT"
```
Comment (anwar6953) [inline cdap-ui/templates/apps/predefined/NaiveBayesTrainer.json]: should bump

File: cdap-ui/templates/apps/predefined/LogsAggregateDataPipeline.json (hunk 2 above)
Comment (ajainarayanan) [inline cdap-ui/templates/apps/predefined/LogsAggregateDataPipeline.json]: This should be ```3.5.0-SNAPSHOT``` I believe

File: cdap-ui/templates/apps/predefined/NaiveBayesTrainer.json (hunk 3 above)
Comment (ajainarayanan) [inline cdap-ui/templates/apps/predefined/NaiveBayesTrainer.json]: Same here.

**Comment by anwar6953** [comment]
If the conflicts were only in the .json template files and you took the release/3.4 version, then remainder LGTM.

File: cdap-ui/templates/apps/predefined/EventClassifier.json (hunk 1 above)
Comment (chtyim) [inline cdap-ui/templates/apps/predefined/EventClassifier.json]: hum.. those a newly added json. Will do a grep.

**Comment by chtyim** [comment]
Only json has conflicts.

**Pull request merged**